js/src/dropdown.js

```diff
@@ -1,3 +1,5 @@
+/* global Popper */
```

Comment:
Each module is compiled by Babel and grunt-contrib-concat concatenate each files in boostrap.js and at the end we use uglify for minify our JS.

Import and export are native in ES6